src/plugins/checkboxes/checkboxes-en.hbs

```diff
@@ -0,0 +1,232 @@
+---
+{
+	"title": "Checkbox and radio button - Provisional feature",
+	"language": "en",
+	"category": "Style",
+	"description": "Working example - provisional feature",
+	"tag": "chckbxrdio",
+	"parentdir": "checkboxes",
+	"altLangPrefix": "checkboxes",
+	"dateModified": "2020-05-11"
+}
+---
+{{>alert-provisional-demos}}
+
+<section role="group" class="provisional">
+	<h2>Example 1</h2>
+	<fieldset class="chkbxrdio-grp wb-checklist">
+		<legend>
+		To qualify, you must meet <strong>all</strong> of the following conditions: <small class="text-muted">(checkboxes + implicit label, large box)</conditions:small>
+		</legend>
+		<ul class="form-group list-unstyled lst-spcd-2">
+		<li class="checkbox">
+			<label>
+			<input type="checkbox" id="cond10">
+				<span>You have lost or will lose your employment or self-employment income for at least half of the 4-week period for which you are applying</span>
+			</label>
+		</li>
+		<li class="checkbox">
+			<label>
+			<input type="checkbox"
+					id="cond11">
+				<span>You are a resident of Canada</span>
+			</label>
+		</li>
+		<li class="checkbox">
+			<label>
+			<input type="checkbox"
+					id="cond12">
+				<span>You are 15 years of age or older when you apply</span>
```

Comment:
Do you need the span here even though it is implicit?

Reply:
The `span` is required for implicit labels not for explicit label. It makes it easier to style a child element of a checked element. Will probably use a plugin approach to support implicit labels for checkboxes and radio buttons.